apps/perms/utils/asset_perm.py

```diff
@@ -42,12 +40,6 @@ def _asset(self):
 
     def validate_permission(self, account_name, protocol):
         with tmp_to_org(self.asset.org):
-            if self.user.is_superuser:
-                account = self.asset.accounts.all().active().get(name=account_name)
-                account.actions = ActionChoices.all()
-                account.date_expired = timezone.now() + timezone.timedelta(days=365)
-                return account
-
             protocols = self.get_permed_protocols_for_user(only_name=True)
             if 'all' not in protocols and protocol not in protocols:
                 return None
```

Comment:
The provided code is from Django application and includes functions to fetch an asset's permissions. Potential issues include variable names using underscores ('_asset') which can be confusing; also there might be more specific naming practices used elsewhere in the project that should follow Python naming conventions (such as snake_case). In general, this section may need some adjustments depending on how it fits alongside other parts of the application.

Additionally, you could consider reorganization if necessary to make the code more organized or clearer in its purpose without breaking existing functionality or adding unnecessary complexity.